cost-analyzer/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
-appVersion: "1.56.0"
+appVersion: "1.57.0"
```

Comment:
We typically handle bumping this version when we cut new releases-- you can leave this as 1.56.0 and this will we bumped when we cut a new release.